Services/CO.CDP.Forms.WebApi.Tests/UseCase/UpdateFormSectionAnswersUseCaseTest.cs

```diff
@@ -148,8 +148,6 @@ public async Task Execute_ShouldCreateNewSharedConsent_WhenNoDraftSharedConsentE
         var command = (formId: section.Form.Guid, sectionId: section.Guid, answerSetId,
             organisationId: organisation.Guid, answers);
 
-        GivenSharedConsentDoesNotExist(section.Form.Guid, organisation.Guid);
-
```

Comment:
Why's this removed?

Reply:
When refactoring DatabaseFormRepository to move share methods to DatabaseShareCodeRepository, I noticed that UpdateFormSectionAnswersUseCase now uses GetSharedConsentDraftWithAnswersAsync, making the call to GetSharedConsentDraftAsync redundant.